lab10.md

```diff
@@ -2,6 +2,13 @@
 
 <img src="https://github.com/openfaas/media/raw/master/OpenFaaS_Magnet_3_1_png.png" width="500px"></img>
 
+Before starting this lab, create a new folder for your files. As this lab builds on an earlier lab make a copy of lab5:
+
+```
+$ cp -r lab5 lab10 \
```

Comment:
What if they didn't do lab5? What folder do we expect them to be in when then run this?

Reply:
They need to do lab5 first anyway, as lab10 uses the issue-bot.